DataStructures/SortedLinkedList/SortedLinkedList.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DataStructures.SortedLinkedList
+{
+    /// <summary>
+    /// TODO.
+    /// </summary>
+    /// <typeparam name="T">TODO. 2.</typeparam>
+    public class SortedLinkedList<T>
```

Comment:
Codacy found an issue: ['T' is not used in the class.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6332606)